### PR TITLE
Added missing Trust patch request

### PR DESCRIPTION
**Pull request by tomijaga**
Resolves: #1657

Account Number:
8de30226230c35bbc1ce4a63c62a7b9c86bf0ce21fc7bc1a984b7884a9f88782

### CODE REVIEW

File: src/containers/BankApi/BankApiValidators/index.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, {FC} from 'react';
 import {NavLink} from 'react-router-dom';
 
-import {DocContainer, DocEndpoint, QueryParamsOffsetAndLimit, RequestResponseSnippet} from 'components';
+import {DocContainer, DocEndpoint, QueryParamsOffsetAndLimit, RequestResponseSnippet, TableParams} from 'components';
 
 const BankApiValidators: FC = () => {
   return (
```
Comment (jamessspanggg) [inline src/containers/BankApi/BankApiValidators/index.tsx:7]: can u update the `lastUpdatedDate`?

File: src/containers/ConfirmationValidatorApi/ConfirmationValidatorApiBanks/index.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React, {FC} from 'react';
 
-import {DocContainer, DocEndpoint, QueryParamsOffsetAndLimit, RequestResponseSnippet} from 'components';
+import {DocContainer, DocEndpoint, QueryParamsOffsetAndLimit, RequestResponseSnippet, TableParams} from 'components';
 
 const ConfirmationValidatorApiBanks: FC = () => {
```
Comment (jamessspanggg) [inline src/containers/ConfirmationValidatorApi/ConfirmationValidatorApiBanks/index.tsx:5]: can update the `lastUpdateDate` as well

File: src/containers/ConfirmationValidatorApi/ConfirmationValidatorApiValidators/index.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React, {FC} from 'react';
 
-import {DocContainer, DocEndpoint, QueryParamsOffsetAndLimit, RequestResponseSnippet} from 'components';
+import {DocContainer, DocEndpoint, QueryParamsOffsetAndLimit, RequestResponseSnippet, TableParams} from 'components';
 
 const ConfirmationValidatorApiValidators: FC = () => {
```
Comment (jamessspanggg) [inline src/containers/ConfirmationValidatorApi/ConfirmationValidatorApiValidators/index.tsx:5]: here too

File: src/containers/PrimaryValidatorApi/PrimaryValidatorApiBanks/index.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React, {FC} from 'react';
 
-import {DocContainer, DocEndpoint, QueryParamsOffsetAndLimit, RequestResponseSnippet} from 'components';
+import {DocContainer, DocEndpoint, QueryParamsOffsetAndLimit, RequestResponseSnippet, TableParams} from 'components';
```
Comment (jamessspanggg) [inline src/containers/PrimaryValidatorApi/PrimaryValidatorApiBanks/index.tsx:3]: here

File: src/containers/PrimaryValidatorApi/PrimaryValidatorApiValidators/index.tsx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import React, {FC} from 'react';
 
-import {DocContainer, DocEndpoint, QueryParamsOffsetAndLimit, RequestResponseSnippet} from 'components';
+import {DocContainer, DocEndpoint, QueryParamsOffsetAndLimit, RequestResponseSnippet, TableParams} from 'components';
```
Comment (jamessspanggg) [inline src/containers/PrimaryValidatorApi/PrimaryValidatorApiValidators/index.tsx:3]: here

**Review by jamessspanggg — COMMENTED** [review]
hi @zinoadidi can you help verify these updates for the docs?

**Comment by zinoadidi** [comment]
doc updates look good @jamessspanggg also started it locally, updates are in there
image

**Comment by Rajputusman** [comment]
**QA Passed**

**Review by jamessspanggg — APPROVED** [review]

**Pull request merged**